### ISSUE TITLE
Discussion--Consolidating Google Analytics

### ISSUE THREAD

**Issue by scharlottej13** [dask/community#243]
We're currently tracking google analytics separately for a number of pages (i.e. docs, dask-ml, dask examples). Are we OK with consolidating these under one google analytics view in the dask-sphinx-theme?

For context, this issue was spurred by https://github.com/dask/dask-examples/pull/219

cc @mrocklin @ian-r-rose @jrbourbeau

**Comment by mrocklin** [comment]
Nope.  Not yet.  Although we'd also like to look at style this week if we
have time
